t/integration/test_inspect.py

```diff
@@ -117,7 +117,7 @@ def test_active(self, inspect):
                     'exchange': '',
                     'routing_key': 'celery',
                     'priority': 0,
-                    'redelivered': False
+                    'redelivered': ANY,
```

Comment:
I'm not sure why these tests stopped passing. AFAICT the `redelivered` seems to be set to `None` implying that we "don't know" (or don't bother to default to `False`) if a task has been redelivered or not. These tests /probably/ (??) don't care about that so changing this to an `ANY` seems like it is okay.

I'd like for someone a bit more familiar with redelivery logic to think about this if possible. I just don't know what we should be expecting here and why it's not `False` like it must have been in the past.

Reply:
`redelivered` is set when the broker redelivers the message (at least with AMQP brokers).
I honestly don't have an idea what has changed. Do you mind debugging this?

Reply:
I'll see if I can find some time later this week